src/main/java/org/gridsuite/directory/server/DirectoryService.java

```diff
@@ -319,4 +319,13 @@ private boolean isPrivateForNotification(UUID parentDirectoryUuid, boolean isCur
     public Flux<ElementAttributes> getElementsAttribute(List<UUID> ids) {
         return Flux.fromStream(() -> directoryElementRepository.findAllById(ids).stream().map(e -> toElementAttributes(e, 0)));
     }
+
+    public Mono<Void> sendUpdateTypeNotification(UUID elementUuid, String userId) {
```

Comment:
Maybe rename this by `sendUpdateDirectoryNotification`.
Which is consistent with related code. We can't know why it's about type here.
Then it's ok to call from explore-server a directory API which ask to emit a UPDATE_DIRECTORY notification.

Reply:
I'm ok with that. Change done.